src/database/translate-operations.ts

```diff
@@ -0,0 +1,373 @@
+                Promise.all(
+                  Object.entries(value.airnode).flatMap(([appType, nextLevelDown]) =>
+                    Promise.all(
+                      Object.entries(nextLevelDown)
+                        .filter(([_cloudType, airnodeNested]) => airnodeNested?.ois)
+                        .flatMap(async ([cloudType, airnodeNestedSource]) => {
+                          const airnodeNested = airnodeNestedSource as Config;
+
+                          await Promise.all(
+                            airnodeNested.ois.map(async (ois) => {
+                              const result = await prisma.oIS.create({
+                                data: {
+                                  provider: { connect: { name: createdProvider.name } },
+                                  title: ois.title,
+                                  version: ois.version,
+                                  ois: airnodeNested.ois,
+                                },
+                                select: { id: true },
+                              });
+                              return { id: result.id };
+                            })
+                          );
+
+                          const deployedAt = new Date(
+                            parseInt(`20${key.substring(0, 2)}`),
+                            parseInt(key.substring(2, 4)) - 1,
+                            parseInt(key.substring(4, 6)),
+                            parseInt(key.substring(7, 9)),
+                            parseInt(key.substring(9, 11))
+                          );
+
+                          return {
+                            applicationType:
+                              appType === 'airnode' ? ApplicationType.Airnode : ApplicationType.Airseeker,
+                            appVersion: airnodeNested.nodeSettings.nodeVersion,
+                            deployedAt,
+                            cloudType: cloudType.toLowerCase() === 'GCP' ? CloudType.GCP : CloudType.AWS,
+                            providerName: createdProvider.name,
+                            config: airnodeNested,
+                          };
+                        })
+                    )
+                  )
+                )
+              )
+            )
+          )
+            .flat(10)
+            .map((airnode) =>
+              prisma.deployment.create({
+                data: { ...airnode, chainApiReference: undefined },
+              })
+            );
+        })
+      );
+    })
+  );
+
+  await Promise.all(
+    Object.values(operations.beaconSets).map(async (beaconset) => {
+      const chainConfigs = await Promise.all(
+        Object.entries(beaconset.chains).map(async ([chainName, chain]) => {
+          return prisma.chainConfiguration.create({
+            data: {
+              chain: {
+                connect: (await prisma.chainInfrastructure.findFirst({
+                  where: { name: chainName },
+                  select: { name: true },
```

Comment:
Shouldn't we use getId as well here and connect with the id instead of the name?

Reply:
mmm, this is an interesting question:

`chain` here refers to ChainInfrastructure. In ChainInfrastructure both the chain's name and its id are unique, so either can be used as a key, although of course we want to mark one as primary. I made the name the PK, but perhaps for consistency the chain's id should be the PK. Yes, I think that's a good idea 💪 
```
model ChainInfrastructure {
  name               String               @id // The name of the chain this record describes
  fullName           String               @unique // The full, human-readable name
  decimalPlaces      Int // The number of decimal places used when displaying this chain's native token
  id                 Int                  @unique // The chainId of this chain
```